### PR TITLE
milestone 3 query multiple table

### PR DESCRIPTION
**Pull request by bronnie313**
## Vet clinic database queries multiple tables 
- Created tables owners and species
- Modified animals' tables and dropped the species column from the animals' table, added species_id and owner_id and referenced them using the foreign key
- inserted the owner's info into the owner's table and the species' info into the species table
- modified the animals' table to include the species_id values and the owner_id values
- add the insertions and modifications to the data.sql file
- Wrote the queries using JOIN  and added the quests in the queries.sql file
- below are the screenshots:
animals for melody
pokemon join
ALL OWNERS
species count
join jennifer orwell
join escape 0
most owned
melody pond who

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -37,7 +37,9 @@ SELECT * FROM animals JOIN species ON animals.species_id = species.id WHERE spec
 SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id ORDER BY owners.full_name;
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
-SELECT owners.full_name FROM owners JOIN animals ON owners.id = animals.owner_id GROUP BY owners.full_name ORDER BY COUNT(*) DESC LIMIT 1;
+SELECT owners.full_name FROM owners
+JOIN animals ON owners.id = animals.owner_id 
+GROUP BY owners.full_name ORDER BY COUNT(*) DESC LIMIT 1;
```
Comment (tresorsawasawa) [inline queries.sql:45]: Screenshot from 2023-07-14 17-20-40
- [ ] `data.sql` file: As required for the project, kindly make sure you have added all data required in the the `owners table` as right now you miss the following one
  - Melody Pond 77 years old.

Comment (tresorsawasawa) [inline queries.sql:45]: Screenshot from 2023-07-14 17-25-10
- [ ] `data.sql` file: As I mentioned in the previous previous comment and for the project requirements, after you have added `Melody Pond 77 years old` in the owners table, kindly ensure that you also assign him to the correct animals he owns as:
  - Melody Pond owns `Charmander`, `Squirtle`, and `Blossom` animals.

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @bronnie313 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @bronnie313, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**